core/src/main/java/org/mule/runtime/core/api/scheduler/SchedulerService.java

```diff
@@ -125,4 +126,11 @@
    */
   Scheduler customScheduler(SchedulerConfig config, int queueSize);
 
+  /**
+   * Provides a read-only view of all currently active {@link Scheduler}s created through this service.
+   * 
+   * @return all currently active {@link Scheduler}s.
+   */
+  List<Scheduler> getActiveSchedulers();
```

Comment:
Is "Active" prefix needed?  Does scheduler service maintain non-active schedulers?